src/components/SecondaryPanes.js

```diff
@@ -45,6 +51,39 @@ const SecondaryPanes = React.createClass({
 
   displayName: "SecondaryPanes",
 
+  renderBreakpointsToggle() {
+    const { toggleAllBreakpoints, breakpoints,
+            breakpointsDisabled, breakpointsLoading } = this.props;
+    const boxClassName = "breakpoints-toggle";
+
+    if (breakpoints.size == 0 || breakpointsLoading) {
+      return dom.input({
+        type: "checkbox",
+        className: boxClassName,
+        disabled: true,
+        title: L10N.getStr("breakpoints.disable")
+      });
+    }
+
+    return dom.input({
+      type: "checkbox",
+      className: boxClassName,
+      onClick: () => toggleAllBreakpoints(!breakpointsDisabled),
+      defaultChecked: true,
+      ref: (input) => {
+        if (input != null) {
+          const isIndeterminate = !breakpointsDisabled &&
+            breakpoints.some(x => x.disabled);
+          input.indeterminate = isIndeterminate;
+          // checked=true would override the indeterminate value
+          input.checked = !breakpointsDisabled && !isIndeterminate;
```

Comment:
From what I know about the indeterminate property, it isn't possible to set it in HTML - only in JavaScript.
`checked` on the other hand could be a standard prop as you mention but it depends on the indeterminate status (we don't want to override the value by setting checked to true). But you're right that `checked` could be set as a prop - but we would need to check the indeterminate condition there too...